coalib/bears/requirements/PythonRequirement.py

```diff
@@ -25,6 +25,41 @@ def multiple(cls, *args):
         Creates a tuple of multiple ``pip`` requirements, consisting of
         either tuples or strings.
 
+        You should use the ``multiple`` method if you have more
+        requirements from the same manager. This can receive both tuples of
+        strings, in case you want a specific version, or a simple string, in
+        case you want the latest version to be installed.
+
+        This is the case where you would provide strings only, to install the
+        latest version automatically:
+
+        >>> REQUIREMENTS = PythonRequirement.multiple(
+        ...                                     'coala_decorators','setuptools')
+
+        And if you choose to mix them, specifying version for some and for some
+        not:
+
+        >>> REQUIREMENTS = PythonRequirement.multiple(
+        ...                         'coala_decorators',('setuptools', '19.2'))
+
+        In case you provide too many arguments into the tuple, an error will be
+        raised:
+
+        >>> REQUIREMENTS = PythonRequirement.multiple(
+        ...             'coala_decorators',('setuptools', '19.2', 'colorama'))
+        Traceback (most recent call last):
+        ...
+        TypeError: The tuple must have 2 elements.
+
+        The same would happen in case you provide something different than a
+        string or a tuple:
+
+        >>> x = [1, 2, 3, 4]
+        >>> REQUIREMENTS = PythonRequirement.multiple(x)
+        Traceback (most recent call last):
+        ...
+        TypeError: The arguments need to be tuples or strings.
+
```

Comment:
this should be moved into the commit which add `multiple`.
As this is quite big, maybe move the adding of `multiple` also into a separate commit?